dev/org.eclipse.codewind.core/binaries/README.txt

```diff
@@ -0,0 +1,9 @@
+Usage: ./pull.sh
+
+The CLI version is the latest available from master.
+You may override the branch with CW_CLI_BRANCH in the env.
+The download will be skipped if the version on disk matches.
+
+The Appsody version is hard-coded into pull-appsody.sh.
```

Comment:
my readme is not accurate, i failed to update it after some changes -

- the scripts are `appsody-pull` and `cli-pull`
- the script does not hard code an appsody version, it will error out if the var is not set